lessons/2_more_strings_and_functions/lesson.py

```diff
@@ -325,7 +369,9 @@ def inner_func():
 # Based on previous examples we should be able to predict what will happen if we print one of these variables
 # without the bool() typecast function call (i.e. print(zero) will print 0). The bool() typecast call will convert
 # these values into either True or False. Before reading the next paragraph, what do you predict the truthieness
-# value of each of these variables will be? 
+# value of each of these variables will be?
+
+# # so... i have no idea.
 
 # If you predicted that zero, negative and empty would be False and positive and non_empty would be True, then you are
```

Comment:
OOPS! I have an error here. This is why I need to proofread better. Okay, I'm so sorry. negative numbers evaluate to Truthy, not to Falsey. So bool(negative) > True. So so sorry about that error. @madeleinehhhh It is only zero that evaluates to false.